client/src/App.tsx

```diff
@@ -31,8 +31,8 @@ interface StateProps {
 
 type Props = StateProps & OwnProps;
 
-class App extends Component<Props, {}> {
-  public async componentWillMount(): Promise<void> {
+class App extends Component<Props> {
+  public async componentDidMount(): Promise<void> {
```

Comment:
This should be an actual Apollo query or at a minimum an async effect

Reply:
What do you mean by this comment? I changed `componentWillMount` to `componentDidMount` as that lifecycle method is deprecated now.

Reply:
Getting the user info should use Apollo instead of manually calling the GraphQL API (this was from the very early days of Bolt before I decided to use Apollo)

Reply:
Ohh huh I didn't even realize that lol I can change it rn

Reply:
I just pushed a commit for this